src/server/game/Entities/Creature/Creature.h

```diff
@@ -62,7 +62,6 @@ class TC_GAME_API Creature : public Unit, public GridObject<Creature>, public Ma
 {
     public:
         explicit Creature(bool isWorldObject = false);
-        virtual ~Creature();
```

Comment:
deleting virtual destructor???

Reply:
It's virtual :-)

Reply:
i see

Reply:
https://www.quantstart.com/articles/C-Virtual-Destructors-How-to-Avoid-Memory-Leaks

Reply:
`Unit::~Unit()` is virtual